js/fieldmanager.js

```diff
@@ -182,7 +182,9 @@ $( document ).ready( function () {
 	// Handle remove events
 	$( document ).on( 'click', '.fmjs-remove', function( e ) {
 		e.preventDefault();
-		fm_remove( $( this ) );
+		if (confirm('Do you want to delete?')){
```

Comment:
We're not going to move forward with this change, but I'll point out a couple things for the sake of peer code review: this string would need to be internationalized, and the code formatting (whitespace) would need to match the rest of the file.